dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/trackedentity/DefaultTrackedEntityInstanceService.java

```diff
@@ -923,6 +923,11 @@ public boolean trackedEntityInstanceExistsIncludingDeleted( String uid )
 
     private boolean isLocalSearch( TrackedEntityInstanceQueryParams params, User user )
     {
+        if ( params.isOrganisationUnitMode( CAPTURE ) )
```

Comment:
would we also not need to check wether the SELECTED org unit is the capture org unit?